pages/index.js

```diff
@@ -68,17 +110,43 @@ export default function HomePage({ onAddEmotionEntry }) {
           On a scale from 0 to 100, how tense do you feel in this moment?
         </StyledLabel>
         <StyledInput
+          aria-label="Adjust tension level between 0 and 100"
           id="tension-level"
           name="tensionLevel"
           type="range"
           defaultValue={0}
           max={100}
+          onChange={(event) => setTension(event.target.value)}
         />
         <StyledWrapper>
           <StyledSpan>0</StyledSpan>
           <StyledSpan>100</StyledSpan>
         </StyledWrapper>
-        <StyledButton type="submit">Save</StyledButton>
+        {!show && <p>{tension}</p>}
+        <StyledButton type="submit" onClick={() => setShow(!show)} $show={show}>
+          Save
+        </StyledButton>
+
+        {show && (
+          <StyledMessage>Your entry was successfully saved!</StyledMessage>
+        )}
+        <StyledButtonWrapper $show={show}>
+          <StyledBackButton
+            type="reset"
+            value={"Done"}
+            onClick={() => {
+              setShow(!show);
+              setTension("0");
+            }}
+          ></StyledBackButton>
+          <StyledLink
+            $actionButton
+            href={{ pathname: "/create", query: { id: id } }}
+            forwardedAs={`/create`}
+          >
+            Add more details
+          </StyledLink>
```

Comment:
As mentioned before. Please check your intention and read the handout for other possible more readable approaches. 🙂